en/administration/licenses.md

```diff
@@ -0,0 +1,112 @@
+---
+id: licenses
+title: Licenses
+---
+
+## How can I obtain a license?
+
+* You can request your token for the [free IT-100 edition](../getting-started/IT100.html) on our website.
+* If you have purchased a license, request your license files from our support team.
+
+## Types of licenses
+
+According to your [Centreon edition](https://www.centreon.com/en/editions/), your license can be:
+- offline: uses one or several license files
+- online: uses a token. Your Centreon platform must be connected to the internet.
+
+## Which modules require a license?
+
+The following modules need to be installed separately and require a valid license.
+
+- [Plugin Packs](https://docs.centreon.com/current/en/monitoring/pluginpacks.html#installation)
+- [Autodiscovery](https://docs.centreon.com/current/en/monitoring/discovery/installation.html)
+- [Anomaly detection](https://docs.centreon.com/current/en/monitoring/anomaly-detection.html)
+- [BAM](https://docs.centreon.com/current/en/service-mapping/install.html)
+- [MAP](https://docs.centreon.com/current/en/graph-views/install.html)
+- [MBI](https://docs.centreon.com/current/en/reporting/introduction.html)
```

Comment:
I think you should put either all the links to the module introduction pages (best idea I think) or all the links to the installation pages
But it's weird to have only one link to introduction and others to installation